src/components/VideoPlayerContexts/VideoPopoverMenuContext.tsx

```diff
@@ -18,13 +19,14 @@ function VideoPopoverMenuContextProvider({children}: ChildrenProps) {
     const [currentPlaybackSpeed, setCurrentPlaybackSpeed] = useState<PlaybackSpeed>(CONST.VIDEO_PLAYER.PLAYBACK_SPEEDS[2]);
     const {isOffline} = useNetwork();
     const isLocalFile = currentlyPlayingURL && CONST.ATTACHMENT_LOCAL_URL_PREFIX.some((prefix) => currentlyPlayingURL.startsWith(prefix));
+    const playerRef = useRef<VideoWithOnFullScreenUpdate | null>(null);
```

Comment:
Can we give it a more meaningful name? IE: videoPopoverMenuRef
```suggestion
    const videoPopoverMenuPlayerRef = useRef<VideoWithOnFullScreenUpdate | null>(null);
```